Library/Homebrew/extend/ENV/super.rb

```diff
@@ -176,6 +176,8 @@ def determine_library_paths
     PATH.new(
       keg_only_deps.map(&:opt_lib),
       HOMEBREW_PREFIX/"lib",
+      "#{MacOS.sdk_path}/usr/lib",
+      (compiler == :llvm_clang ? Formula["llvm"].opt_lib.to_s : ""),
```

Comment:
Don't use a ternary here; split out this logic by building the `PATH` differently.

Reply:
I'm guessing that a ternary also shouldn't be used for separate CLT/Xcode SDK include paths?

Reply:
Correct, thanks.